Controller/ChangePasswordFOSUser1Controller.php

```diff
@@ -33,7 +33,7 @@ public function changePasswordAction()
     {
         $user = $this->getUser();
         if (!is_object($user) || !$user instanceof UserInterface) {
-            $this->createAccessDeniedException('This user does not have access to this section.');
+            throw $this->createAccessDeniedException('This user does not have access to this section.');
```

Comment:
We're calling this method any three different classes at the moment:
https://github.com/sonata-project/SonataUserBundle/search?utf8=%E2%9C%93&q=createAccessDeniedException